chipsec/helper/linux/linuxhelper.py

```diff
@@ -1178,7 +1178,7 @@ def decompress_file( self, CompressedFileName, OutputFileName, CompressionType )
     #
     def get_threads_count ( self ):
         import subprocess
-        return int(subprocess.check_output("grep -c process /proc/cpuinfo", shell=True))
+        return int(subprocess.check_output("egrep -c ^process /proc/cpuinfo", shell=True))
```

Comment:
Why egrep vs grep?

Reply:
not really needed as the regex is pretty simple. I somehow think we should use the info from MSRs rather than parsing a tool.